internal/pkg/service/stream/mapping/table/column/renderer.go

```diff
@@ -32,7 +33,13 @@ func (r *Renderer) CSVValue(c Column, ctx *receivectx.Context) (string, error) {
 	case Headers:
 		return json.EncodeString(ctx.HeadersMap(), false)
 	case UUID:
-		id, err := uuid.NewV7()
+		id, err := uuid.NewGenWithOptions(
+			uuid.WithEpochFunc(
+				func() time.Time {
+					return ctx.Now
+				},
+			),
+		).NewV7()
```

Comment:
@jachym-tousek-keboola you are now creating new generator for each column value, so a benchmark is needed, to check peformance impact :) or keep the old :heavy_check_mark:  approved version.

This is part of critical code path for high throughput.

Reply:
I prefer faster version, the value is always partially random anyway, so using `now` has no benefits even for tests.